Robitaille/template/buggy_function.py

```diff
@@ -23,10 +23,5 @@ def angle_to_sexigesimal(angle_in_degrees, decimals=3):
     hours_num = angle_in_degrees*24/180
     hours = math.floor(hours_num)
 
-    min_num = (hours_num - hours)*60
```

Comment:
I'm lazy and didn't actually think that hard about what this does.  But can you confirm you really meant to delete this with no other replacement?